linkerd.hcl

```diff
@@ -0,0 +1,30 @@
+description = "Linkerd is an ultralight, security-first service mesh for Kubernetes. Linkerd adds critical security, observability, and reliability features to your Kubernetes stack with no code change required."
+binaries = ["linkerd"]
+test = "linkerd version --client"
+
+platform darwin amd64 {
+    source = "https://github.com/linkerd/linkerd2/releases/download/stable-${version}/linkerd2-cli-stable-${version}-${os}"
+    on unpack {
+        rename { from = "${root}/linkerd2-cli-stable-${version}-${os}" to = "${root}/linkerd" }
+    }
+}
+
+platform darwin arm64 {
```

Comment:
This and the linux block look identical, so you could just move the source+unpack to the top level and have a single override for darwin+amd64 if you prefer